config-example/LOCAL/config.json

```diff
@@ -1,5 +1,6 @@
 {
     "is_local": true,
+    "is_prod": false,
```

Comment:
technically this is not needed as the default is set in `aladdin.sh` if the value extraction fails: https://github.com/fivestars-os/aladdin/blob/1.19.7.14/aladdin.sh#L157-L160

notice that the sample PROD config has the correct setting: https://github.com/fivestars-os/aladdin/blob/1.19.7.14/config-example/CLUSTERPROD/config.json#L6

is there a compelling reason for setting this in this example LOCAL config?

Reply:
@abandali , thanks, just verified, it works fine without this option.